lib/dir.go

```diff
@@ -0,0 +1,60 @@
+package lib
+
+import (
+	"io/ioutil"
+	"os"
+	"path"
+)
+
+// DirCopy copies a whole directory recursively
+func DirCopy(dest string, source string) error {
+
+	var err error
+	var fds []os.FileInfo
+	var srcinfo os.FileInfo
+
+	if srcinfo, err = os.Stat(source); err != nil {
+		return err
```

Comment:
Looks like there's a bunch of filesystem operations here and in a few other functions which don't use errors.Wrapf(), which prevents us from getting stack traces from the errors making things harder to debug. Can you add a Wrapf() from these filesystem operations?

You don't need to add Wrapf() to errors which are already wrapped (aka ones that come from stacker or umoci APIs), just ones that won't include stack trace info.

Reply:
Ok, got it. Thanks

Reply:
Seems like we still need a bunch of Wrapf()s here?

Reply:
Or at least WithStack()